extensions/wikia/Forum/ForumController.class.php

```diff
@@ -172,24 +195,30 @@ public function boardThread() {
 		} else {
 			$displayname = $name;
 			$displayname2 = '';
-			$url = Title::newFromText( $name, $this->wg->EnableWallExt ? NS_USER_WALL : NS_USER_TALK )->getFullUrl();
+			$url =
+				Title::newFromText( $name, $this->wg->EnableWallExt ? NS_USER_WALL : NS_USER_TALK )
+					->getFullUrl();
 		}
 
 		$this->response->setVal( 'username', $name );
 		$this->response->setVal( 'displayname', $displayname );
 		$this->response->setVal( 'displayname2', $displayname2 );
 		$this->response->setVal( 'user_author_url', $url );
 		$this->response->setVal( 'iso_timestamp', $lastReply->getCreatTime( TS_ISO_8601 ) );
-		$this->response->setVal( 'fmt_timestamp', $this->wg->Lang->timeanddate( $lastReply->getCreatTime( TS_MW ) ) );
+		$this->response->setVal( 'fmt_timestamp',
+			$this->wg->Lang->timeanddate( $lastReply->getCreatTime( TS_MW ) ) );
 
 		wfProfileOut( __METHOD__ );
 	}
 
 	public function breadCrumbs() {
 		if ( $this->app->wg->Title->getNamespace() == NS_WIKIA_FORUM_TOPIC_BOARD ) {
 			$indexPage = Title::newFromText( 'Forum', NS_SPECIAL );
-			$path = [ ];
-			$path[] = [ 'title' => wfMessage( 'forum-forum-title' )->escaped(), 'url' => $indexPage->getFullUrl() ];
+			$path = [];
+			$path[] = [
+				'title' => wfMessage( 'forum-forum-title' )->escaped(),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "title" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)